pkg/config/coordinator.go

```diff
@@ -26,6 +26,8 @@ type Coordinator struct {
 	UseSystemdNotifier   bool          `json:"use_systemd_notifier" toml:"use_systemd_notifier" yaml:"use_systemd_notifier"`
 	SystemdNotifierDebug bool          `json:"systemd_notifier_debug" toml:"systemd_notifier_debug" yaml:"systemd_notifier_debug"`
 	IterationTimeout     time.Duration `json:"iteration_timeout" toml:"iteration_timeout" yaml:"iteration_timeout"`
+	EnableRoleSystem     bool          `json:"enable_role_system" toml:"enable_role_system" yaml:"enable_role_system"`
```

Comment:
Add `enable_role_system` and `roles_file` to the configuration documentation for both the coordinator and the router.